.ci/Jenkinsfile

```diff
@@ -2,7 +2,7 @@ pCloud = "colo-research-01"
 gitUrl = null
 gitBranch = null
 gitCommit = null
-pTimeout = '1800' // in seconds
+pTimeout = '3600' // in seconds
```

Comment:
Can you make this timeout just for the docker build, rather than globally? Will need a new variable.